luigi.cfg

```diff
@@ -0,0 +1,8 @@
+[core]
+default-scheduler-host = pyroclast-luigi.gilgi.org
+default-scheduler-port = 80
```

Comment:
Wrong. Also you don't need to set it. You only need to set default-scheduler-url. Luigi will take care of everything else.

Reply:
Oh, gilgi gave me this from his config. Is it bad enough that we need to delete it?

Reply:
Yes, they could potentially conflict later on. KISS (keep it simple stupid)